### DISCUSSION TITLE
sidecar with Kafkaconnect

### DISCUSSION THREAD

**Discussion by ee07dazn** [strimzi/strimzi-kafka-operator#7609] [Q&A] [answered]
Hi

Is it possible to add a sidecar container with Kafkaconnect. I want to use `Cloud SQL Auth proxy` in a sidecar pattern and wondering if i can do this with kafkaconnect, so as to allow one of the connectors to communicate with cloudsql instance in a different gcp project.

Thanks

**Comment by scholzj** [comment, marked as answer]
No, it is not possible. You can probably run it as a separate deployment.

**Reply by ee07dazn** [reply]
Thanks for confirming.
